easybuild/toolchains/iccifortcuda.py

```diff
@@ -38,3 +39,22 @@ class IccIfortCUDA(IccIfort, Cuda):
 
     COMPILER_MODULE_NAME = IccIfort.COMPILER_MODULE_NAME + Cuda.COMPILER_CUDA_MODULE_NAME
     SUBTOOLCHAIN = IccIfort.NAME
+
+    def is_dep_in_toolchain_module(self, name):
+        """Check whether a specific software name is listed as a dependency in the module for this toolchain."""
+        # icc & ifort do not need to be actual dependencies in iccifort module,
+        # since they could also be installed together in a single directory.
+        # Let IccIfort check that.
+        res = IccIfort.is_dep_in_toolchain_module(self, name)
+
+        # Also check for CUDA since this is IccIfortCUDA toolchain
+        # as long as the corresponding $EBROOT* and $EBVERSION* environment variables are defined, it should be OK
+        if not res:
+            if name in 'CUDA':
```

Comment:
@akesandgren This should be `if name == 'CUDA'`, with `in` you're less strict (`CU` would also pass that test)

Reply:
Ah, me and python... too much c&p from the iccifort.py code.